scripts/devaccess/helper_scripts/rhmi_3scale_dump.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+THREESCALE_DUMP_SCRIPT=/usr/local/bin/3scale-dump.sh
+OUTPUT_FILE=3scale-dump.tar
```

Comment:
Just adding a tiny tweak, solely so I can get a new commit/test cycle happening.
```suggestion
THREESCALE_DUMP_SCRIPT="/usr/local/bin/3scale-dump.sh"
OUTPUT_FILE="3scale-dump.tar"
```